apps/src/gamelab/constants.js

```diff
@@ -204,5 +204,35 @@ module.exports.SpritelabReservedWords = [
   'whileRightArrow',
   'whileUpArrow',
   'xLocationOf',
-  'yLocationOf'
+  'yLocationOf',
+  // SpriteLab2Beta.interpreted.js
+  'customText',
+  'animationGroups',
+  'emptyGroup',
+  'thisSprite',
+  'otherSprite',
+  'addToAnimationGroup',
+  'allSpritesWithAnimation',
+  'checkTouching',
+  'countByCostume',
+  'createCollisionHandler',
+  'drawBackground',
+  'keyPressed',
+  'layoutSprites',
+  'makeNewSpriteAnon',
+  'printCustomText',
+  'printText',
+  'removeFromAnimationGroup',
+  'runCallbacks',
+  'runCollisionEvents',
+  'runCollisionEventsExperimental',
+  'runInputEvents',
+  'runLoops',
+  'runSpriteBehaviors',
+  'setBackgroundImage',
+  'singleOrGroup',
+  'spriteClicked',
+  'updateHUDText',
+  'i',
+  'j'
```

Comment:
Seems like we should file a task so we don't keep this here long term.

Reply:
Many of these words will be removed from the list once we move the spritelab library to native (so hopefully very soon)